### PR TITLE
Add Yandex support ( Fixes #201 )

### LINKED ISSUE #201: Add support for Yandex
Adding enhancement of [WebCrawler](http://www.webcrawler.com/), [WolframAlpha](http://www.wolframalpha.com/) and [Yandex](https://www.yandex.com/) to the project.

**Would you like to work on this?**
yes

### PR DESCRIPTION
**Pull request by bhaveshAn**
Fixes #201 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Adding support for Yandex
- 
screenshot from 2017-10-05 18-37-25

### CODE REVIEW

File: app/scrapers/__init__.py [hunk 1]
```diff
@@ -6,13 +6,15 @@
 from bing import Bing
 from yahoo import Yahoo
 from ask import Ask
+from yandex import Yandex
```
Comment (cclauss) [inline app/scrapers/__init__.py]: This is correct but might need to be changed if #227 gets accepted first.

File: app/scrapers/yandex.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    def parseResponse(self, soup):
+        """ Parse the response and return set of urls
+        Returns: urls (list)
+                [[Tile1,url1], [Title2, url2],..]
+        """
+        urls = []
+        for a in soup.findAll('a', {'class': 'link link_theme_normal'}):
+            urls.append({'title': a.getText(), 'link': a.get('href')})
+
```
Comment (cclauss) [inline app/scrapers/yandex.py]: What you have here is correct.  I tend to like list comprehensions but other folks do not.  It is a matter of taste...
```python
        urls = [{'title': a.getText(), 'link': a.get('href')} for a
                in soup.findAll('a', class_='link link_theme_normal')]
```

File: app/templates/index.html [hunk 3]
```diff
@@ -59,7 +59,8 @@
                             <button type="submit" value="bing" class="btn btn-lg  search btn-outline"><img src="{{ url_for('static', filename='images/bing_icon.ico') }}" width="30px" alt="Bing Icon"> Bing</button>
                             <button type="submit" value="duckduckgo" class="btn btn-lg  search btn-outline"><img src="{{ url_for('static', filename='images/duckduckgo_icon.png') }}" width="30px" alt="Duckduckgo Icon"> DuckDuckGo</button>
                             <button type="submit" value="yahoo" class="btn btn-lg  search btn-outline"><img src="{{ url_for('static', filename='images/yahoo_icon.ico') }}" width="30px" alt="Yahoo Icon"> Yahoo</button>
-			    <button type="submit" value="ask" class="btn btn-lg  search btn-outline"><img src="{{ url_for('static', filename='images/ask_icon.ico') }}" width="30px" alt="Ask Icon"> Ask</button>
+			                <button type="submit" value="ask" class="btn btn-lg  search btn-outline"><img src="{{ url_for('static', filename='images/ask_icon.ico') }}" width="30px" alt="Ask Icon"> Ask</button>
```
Comment (cclauss) [inline app/templates/index.html]: A bit of over-indentation?

Comment (bhaveshAn) [reply app/templates/index.html]: @cclauss Done

**Comment by bhaveshAn** [comment]
Resolved conflicts and rebased the commits.

File: app/scrapers/__init__.py [hunk 4]
```diff
@@ -6,13 +6,15 @@
...
 
 scrapers = {
     'g': Google(),
     'b': Bing(),
     'y': Yahoo(),
     'd': Duckduckgo(),
-    'a': Ask()
+    'a': Ask(),
+    'yd': Yandex()
```
Comment (cclauss) [inline app/scrapers/__init__.py]: Same single letter problem as #230

**Comment by mariobehling** [comment]
Thanks. Let's try.

**Pull request merged**